virtualizarr/readers/hdf/hdf.py

```diff
@@ -139,8 +148,8 @@ def _construct_manifest_group(
         non_coordinate_dimesion_vars = HDFVirtualBackend._find_non_coord_dimension_vars(
             group=g
         )
-        drop_variables = list(set(drop_variables + non_coordinate_dimesion_vars))
-        attrs: dict[str, Any] = {}
+        drop_variables = list(set(list(drop_variables) + non_coordinate_dimesion_vars))
+        attrs = HDFVirtualBackend._extract_attrs(g)
```

Comment:
Can you add a comment for why `non_coordinate_dimension_vars` is added to `drop_variables`? Does the use of t`o_virtual_dataset` not help isolate this type of decision from the readers?

Reply:
@maxrjones 👍 I'll add a comment and an xfailing test.  This is just a temporary patch because some of our fixtures generated from xarray tutorial data serialize dimensions to HDF with big endian dtypes that are not yet supported in Zarr v3 https://github.com/zarr-developers/VirtualiZarr/pull/545

Reply:
@maxrjones This commit should address this https://github.com/zarr-developers/VirtualiZarr/pull/542/commits/1d128394b408c034dc71a2cd60e160a1b0dd15dc